origin/tunnel.go

```diff
@@ -516,6 +542,8 @@ func (h *TunnelHandler) ServeStream(stream *h2mux.MuxedStream) error {
 	h.logRequest(req, cfRay, lbProbe)
 	if websocket.IsWebSocketUpgrade(req) {
 		conn, response, err := websocket.ClientConnect(req, h.tlsConfig)
+		h.logger.WithFields(log.Fields{"connectionID": h.connectionID, "status": response.StatusCode}).Info("incrementResponses")
```

Comment:
this pattern, `h.xxx(h.yyy(h.z))` seems like a code smell and suggests that more of the logic be moved from metrics into the TunnelHandler